src/lib/services/proxy-service.ts

```diff
@@ -134,7 +133,7 @@ export class ProxyService {
             token,
         );
 
-        const client = await startUnleash({
```

Comment:
we can not use startUnleash as it will reuse the same unleash instance if the config is the same (based on a hash of the config). We should instantiate a new instance because that is exactly what we want here.